internal/datastore/crdb/readwrite.go

```diff
@@ -267,8 +268,29 @@ func (rwt *crdbReadWriteTXN) WriteNamespaces(newConfigs ...*core.NamespaceDefini
 	return nil
 }
 
-func (rwt *crdbReadWriteTXN) DeleteNamespace(nsName string) error {
-	ctx := datastore.SeparateContextWithTracing(rwt.ctx)
+func (rwt *crdbReadWriteTXN) DeleteNamespaces(nsNames ...string) error {
+	ctx, span := tracer.Start(
+		datastore.SeparateContextWithTracing(rwt.ctx),
+		"DeleteNamespaces",
+		trace.WithAttributes(attribute.StringSlice("names", nsNames)),
```

Comment:
I've been using this sort of "standardized" attribute keys. I think it makes sense to this this to avoid unintended proliferation of keys. WDYT?

https://github.com/authzed/spicedb/blob/114455fcc6f263ef5853fe52be4658de9eb960ca/internal/datastore/common/sql.go#L22-L45